src/plugins/workspace/opensearch_dashboards.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "workspace",
+  "version": "opensearchDashboards",
+  "server": true,
+  "ui": false,
+  "requiredPlugins": [
+    "savedObjects"
```

Comment:
I think savedObjects is one I suggested. Just in case, I would suggest in a follow-up verifying all your requiredPlugins and bundles